recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -0,0 +1,14 @@
+# Extended existing
+FILES_${PN}-bnx2 += " \
+    ${nonarch_base_libdir}/firmware/bnx2/bnx2-mips*.fw \
+    ${nonarch_base_libdir}/firmware/bnx2/bnx2-rv2p*.fw \
+"
+RDEPENDS_${PN}-bnx2 = "${PN}-bnx2-mips"
+
```

Comment:
I would like to request:
```
PACKAGES_prepend = " ${PN}-iwlwifi-pnvm "

LICENSE_${PN}-iwlwifi-pnvm = "Firmware-iwlwifi_firmware"
FILES_${PN}-iwlwifi-pnvm = "/lib/firmware/iwlwifi-*pnvm*"
RDEPENDS_${PN}-iwlwifi-pnvm  = "${PN}-iwlwifi-license"
```
and also adding it to `recipes-core/images/xenclient-ndvm-image.bb`.  This is because the PNVM firmware isn't shipped with the ucode normally, but AX210+ wifi cards need it available.

Reply:
Good idea - thanks.  What do you think about just putting them into ${PN}-iwlwifi-misc?